src/Http/StringHttpAdapter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Aidantwoods\SecureHeaders\Http;
+
+use Aidantwoods\SecureHeaders\HeaderBag;
+
+class StringHttpAdapter implements HttpAdapter
+{
+    private $headers = array();
+
+    /**
+     * Send the given headers, overwriting all previously send headers
+     *
+     * @param HeaderBag $headers
+     * @return void
+     */
+    public function sendHeaders(HeaderBag $headers)
+    {
+        $this->headers = $headers->get();
+    }
+
+    /**
+     * Retrieve the current list of already-sent (or planned-to-be-sent) headers
+     *
+     * @return HeaderBag
+     */
+    public function getHeaders()
+    {
+        return new HeaderBag();
```

Comment:
I know `StringHttpAdapter` isn't really meant to ever retrieve headers as an object, but just to keep this as a good implementation example for others, I guess this should be `return new HeaderBag($this->headers);`, rather than a blank list of headers?